phpunit/functional/PendingReasonTest.php

```diff
@@ -500,7 +500,7 @@ protected function testUpdatesFromNewTimelineItemProvider(): iterable
                 'pendingreasons_id'           => $pending_reason1->getID(),
                 'followup_frequency'          => 3 * DAY_TIMESTAMP,
                 'followups_before_resolution' => 2,
-                // Pending reason is attached to the first followup
+                // Pending reason is attached to the last followup
```

Comment:
I understand the goal of this change, but I still have some doubts about its impact.
In any case, I think it's important to complete the test by also checking which follow-up the pending reason is attached to, perhaps by verifying `last_bump_date` and possibly `bump_count`.